docs/scoring-and-judgment.md

```diff
@@ -3,95 +3,52 @@ id: scoring-and-judgment
 title: Scoring and Judgment
 ---
 
-The scoring system and judgment system in Bemuse focuses on both
-accuracy and combo.
+The scoring system and judgment system in Bemuse focuses on both accuracy and combo.
 
 ## Judgment
 
-When hitting the note, the accuracy of your button press will be judged
-according to this table:
+When hitting the note, the accuracy of your button press will be judged according to this table:
 
-|      Name      | Maximum Offset (ms) | Accuracy Score |
-|:--------------:| -------------------:| --------------:|
-|   METICULOUS!  |                  20 |           100% |
-|     PRECISE!   |                  50 |            80% |
-|      GOOD!     |                 100 |            50% |
-|     OFFBEAT!   |                 200 |             0% |
-|     MISSED!    |                  -- |             0% |
+| Judgment   | Normal | Level5 | Level4 | Level3 | Beginner |
+| ---------- | -----: | -----: | -----: | -----: | -------: |
+| Meticulous |   20ms |   21ms |   22ms |   23ms |     24ms |
+| Precise    |   50ms |   60ms |   70ms |   80ms |    100ms |
+| Good       |  100ms |  120ms |  140ms |  160ms |    180ms |
+| Offbeat    |  200ms |  200ms |  200ms |  200ms |    200ms |
 
-<div class="srcref admonition">
-<p class="first admonition-title">Source code reference</p>
-<ul class="last simple">
-<li>Defined at <a class="reference external" href="https://github.com/bemusic/bemuse/blob/bf96099/src/game/judgments.js#L7-14">src/game/judgments.js:7-14</a> (judgment timegate)</li>
-<li>Defined at <a class="reference external" href="https://github.com/bemusic/bemuse/blob/bf96099/src/game/judgments.js#L51-58">src/game/judgments.js:51-58</a> (judgment weight)</li>
-</ul>
-</div>
+The different timegates are designed to make the game easier for beginners, with a gradual increase in difficulty the harder the song gets. They are divided as follows:
 
-## Scoring
+| Timegate | Song Level |
+| -------- | ---------: |
+| Beginner |        1-2 |
+| Level3   |          3 |
+| Level4   |          4 |
+| Level5   |          5 |
+| Normal   |         6+ |
 
-The player's score is calculated from this formula:
+Based on the above judgement, an accuracy score is given based on the above judgement out of a highest possible score of 555555.
```

Comment:
I think this part is inaccurate...

---

The final score is split into 2 parts:

- 500000 for accuracy
- 55555 for combo bonus

Each combo has an associated score.

| Combo number | Combo score |
| ---:| ---:|
| 0 | 0 |
| 1–22 | 1 |
| 23-50 | 2 |
| 51-91 | 3 |
| 92–160 | 4 |
| 161~∞ | 5 |

Combo bonus is the sum of combo score for each hit, divided by maximum possible value, times 55555.

**Example:** There are 100 notes. 60 notes have been hit, I missed a note, and hit 39 other notes.

- Obtained combo score = 108 for first 60 notes + 56 for 39 other notes = 164
- Maximum possible combo score = 237
- Combo bonus = 164 / 237 * 55555 = 38443